bot/exts/moderation/incidents.py

```diff
@@ -130,6 +138,94 @@ def has_signals(message: discord.Message) -> bool:
     return ALL_SIGNALS.issubset(own_reactions(message))
 
 
+def shorten_text(text: str) -> str:
+    """Truncate the text if there are over 3 lines or 300 characters, or if it is a single word."""
+    original_length = len(text)
+    lines = text.count("\n")
+    # Limit to a maximum of three lines
+    if lines > 3:
+        text = "\n".join(line for line in text.split('\n')[:3])
+    # If it is a single word, then truncate it to 50 characters
+    if text.count(" ") < 1:
```

Comment:
Alternatively find the first occurence of a space so you don't have to keep counting if you do find one.